sys/riscv/riscv/locore.S

```diff
@@ -343,6 +339,8 @@ va:
 #if __has_feature(capabilities)
 	cmove	ct0, cnull
 	cspecialw sscratchc, ct0
+	/* Ensure stdc is null and thus not used */
```

Comment:
```suggestion

	/* Ensure stdc is null and thus not used */
```

Reply:
I didn't want to insert a line break because it's using the ct0 set to null before it, so if it's grouped then that's more obvious

Reply:
Hmm, newlines before comments is just the normal style(9) is all.  Either way is fine though.